x-pack/plugins/canvas/public/expression_types/arg_types/font.js

```diff
@@ -51,10 +52,18 @@ FontArgInput.propTypes = {
 
 FontArgInput.displayName = 'FontArgInput';
 
-export const font = () => ({
+const fontUI = intl => ({
   name: 'font',
-  displayName: 'Text Settings',
-  help: 'Set the font, size and color',
+  displayName: intl.formatMessage({
+    id: 'xpack.canvas.expressionTypes.textSettingsLabel',
```

Comment:
xpack.canvas.expressionTypes => xpack.canvas.argTypes for all ids in this file. cc @maryia-lapata  how do you think?